xdg-ninja.sh

```diff
@@ -30,58 +28,115 @@ fi
 
 unalias -a
 
-HELPSTRING="""\
+#Set ansi color code variables
+set_colors() {
+    case $COLOR in
```

Comment:
I think, `always` option should be added for completeness

Reply:
If `always` was added it would do nothing, or I would have to move all of the variable setting inside of `auto` and `always`, and add a catchall that outputs an error which will increase line-count and make stuff less readable.

Reply:
Do nothing then, and warn user if `COLOR` is not in {`never`, `auto`, `always`}. Also add `always` option to help

Reply:
I also suggest to prefix environment variable names with `XDG_NINJA_`, since variable names like `COLOR`, `DECODER` are pretty common and may mess things up if set globally

Reply:
`NO_COLOR` is actually a standard:
https://no-color.org/

Reply:
> `NO_COLOR` is actually a standard: https://no-color.org/

Ok, don't change it then, but deal with other environment variable names

Reply:
That's the only environment variable the program reads (other than the XDG ones)

Reply:
Isn't `COLOR` read? I also would let user to provide one's own decoder.

Reply:
Nope, none of that is read from env variables, just use flags. I could pretty easily implement a config file in this PR ~~as well ||and make it default to `~/.xdg-ninjarc`.~~

Reply:
> Nope, none of that is read from env variables, just use flags. I could pretty easily implement a config file in this PR ~as well ||and make it default to `~/.xdg-ninjarc`.~

:)

So this is resolved